js/src/product-feed/issues-table-card/issues-table.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Most of this was in the index before. 

Now we just do

` const { data, hasFinishedResolution, page, setPage } = useMCIssues( getActiveIssueType() );`

To get the current issues by type. And render the Paginator and the table data.